src/test/rbd_mirror/test_mock_ImageMap.cc

```diff
@@ -612,7 +612,7 @@ TEST_F(TestMockImageMap, AcquireImageErrorRetry) {
   std::set<std::string> initial_global_image_ids_ack(initial_global_image_ids);
 
   // UPDATE_MAPPING failure
-  expect_add_event(mock_threads);
+   expect_add_event(mock_threads);
```

Comment:
why this space?

Reply:
not intentional, obviously. thanks for noticing.